### ISSUE TITLE
Modal close button disappears in forced-color mode

### ISSUE THREAD

**Issue by SebinSong** [okTurtles/group-income#2440]
### Problem

<img src='https://github.com/user-attachments/assets/7b76f303-a7d0-44d6-9683-2703278639b6' width='480'>

Modal close button is there but not visible when forced-color mode is enabled.

### Solution
Fix it.
